### PR TITLE
2.0 quickstarts/scanner/*

### PR DESCRIPTION
**Pull request by joecheuk**

### CODE REVIEW

File: _docs/quickstarts/scanner/default_rules.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+order: 106
+---
+
+# {{ page.title }}
+
+Forseti Scanner has default rules that create a [violation]({% link _docs/quickstarts/scanner/policy-violations.md %}) when their conditions are met.
+
+* BigQuery
+  * Datasets should not be public.
+  * Datasets should not be accessible by users who are @gmail.com.
```
Comment (mirons-google) [inline _docs/quickstarts/scanner/default_rules.md]: nit: by users who's email address matches '@gmail.com'

File: _docs/quickstarts/scanner/default_rules.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  * Datasets should not be accessible by groups who are @gmail.com.
+
+* Blacklist
+  * IP address of any GCP instances should not be listed on the [emergingthreats](https://rules.emergingthreats.net/fwrules/emerging-Block-IPs.txt) website
```
Comment (mirons-google) [inline _docs/quickstarts/scanner/default_rules.md]: The IP address

File: _docs/quickstarts/scanner/default_rules.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+* Cloud Storage (_note: this is NOT the same as Cloud Storage IAM policies!_)
```
Comment (mirons-google) [inline _docs/quickstarts/scanner/default_rules.md]: Cloud Storage (legacy ACL policies)

File: _docs/quickstarts/scanner/descriptions.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
+---
+title: Scanner Specifications
```
Comment (mirons-google) [inline _docs/quickstarts/scanner/descriptions.md]: nit: This Title should should match Scanner quickstarts in someway.

File: _docs/quickstarts/scanner/descriptions.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+order: 102
+---
+
+# {{ page.title }}
+
+This page describes the Forseti scanners that are available, how they work, and
+why they're important. You can configure Scanner to execute multiple scanners in
+the same run. Learn about [configuring
+Scanner]({% link _docs/configure/scanner/index.md %}).
```
Comment (mirons-google) [inline _docs/quickstarts/scanner/descriptions.md]: weird line split.

File: _docs/quickstarts/scanner/descriptions.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+
+VM instances with external IP addresses communicate with the outside world.
+If they are compromised, they could appear in various 
+blacklists and could be known as malicious, such as for sending spam, 
+hosting Command & Control servers, and so on. The blacklist scanner audits
+all of the VM instances in your environment and determines if any VMs
+with external IP addresses are on a specific blacklist you've configured.
+
+For examples of how to define scanner rules, see the
+[`blacklist_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/rules/blacklist_rules.yaml)
```
Comment (mirons-google) [inline _docs/quickstarts/scanner/descriptions.md]: feel free to let URL lines extend pass 80

File: _docs/quickstarts/scanner/descriptions.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+Cloud SQL scanner supports a blacklist mode, to ensure unauthorized users don't
+gain access to your Cloud SQL instances.
+
+For examples of how to define scanner rules for your Cloud SQL instances, see
+the
+[`cloudsql_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/rules/cloudsql_rules.yaml)
+rule file.
+
+## Firewall Rules scanner
+Network firewall rules protects your network & organization by only allowing
```
Comment (mirons-google) [inline _docs/quickstarts/scanner/descriptions.md]: Be consistent with leaving a new line or not leaving a new line. It seems you are leaving a new line between the headings and the content.

Comment (joecheuk) [reply _docs/quickstarts/scanner/descriptions.md]: Updated.

File: _docs/quickstarts/scanner/descriptions.md (hunk 6 above)
Comment (joecheuk) [reply _docs/quickstarts/scanner/descriptions.md]: Updated.

File: _docs/quickstarts/scanner/descriptions.md (hunk 5 above)
Comment (joecheuk) [reply _docs/quickstarts/scanner/descriptions.md]: Updated.

File: _docs/quickstarts/scanner/default_rules.md (hunk 3 above)
Comment (joecheuk) [reply _docs/quickstarts/scanner/default_rules.md]: Updated.

File: _docs/quickstarts/scanner/default_rules.md (hunk 2 above)
Comment (joecheuk) [reply _docs/quickstarts/scanner/default_rules.md]: Updated.

File: _docs/quickstarts/scanner/default_rules.md (hunk 1 above)
Comment (joecheuk) [reply _docs/quickstarts/scanner/default_rules.md]: Updated.

File: _docs/quickstarts/scanner/descriptions.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+VM instances with external IP addresses expose your environment to an
+additional attack surface area. The instance network interface scanner audits
+all your VM instances in your environment, and determines if any VMs with
+external IP addresses are outside of the trusted networks.
+
+For examples of how to define scanner rules for network interfaces, see the 
+[`instance_network_interface_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/rules/instance_network_interface_rules.yaml)
+rule file.
+
+## KE Version scanner
```
Comment (Tabrarian) [inline _docs/quickstarts/scanner/descriptions.md]: Is this an approved or official name for the scanner? We aren't supposed to abbreviate "Kubernetes Engine" per branding specifications.

File: _docs/quickstarts/scanner/index.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+writes it to Cloud Storage bucket.
+
+Forseti Scanner is different from the Cloud Security Scanner, which does App
+Engine vulnerability scanning. Learn more about
+[Cloud Security Scanner](https://cloud.google.com/security-scanner/).
+
+## Running Forseti Scanner
+
+Forseti Scanner works on a data model, so before you start using Scanner, you'll select a model to use. 
+For information about how to create a model, see Testing Models.
```
Comment (Tabrarian) [inline _docs/quickstarts/scanner/index.md]: Do we have a link for this yet?

File: _docs/quickstarts/scanner/rules.md [hunk 10]
```diff
@@ -0,0 +1,371 @@
...
+  The list of rules for what versions are allowed on nodes.
+  - **major**: The major version that is allowed.
+  - **minor**: Optional, the minor version that is allowed. If not included, all minor versions are allowed.
+  - **operator**: Optional, defaults to =, can be one of (=, >, <, >=, <=). The operator determines
+    how the current version compares with the allowed version. If a minor version is not included,
+    the operator applies to major version. Otherwise it applies to minor versions within a single major version.
+
+### Enabling
+
+To enable the KE inventory, add the following to the inventory section in your forseti_confi.yaml file.
```
Comment (Tabrarian) [inline _docs/quickstarts/scanner/rules.md]: For the following sections, can you confirm if KE is an approved or official module name? Per branding guidelines, we aren't supposed to abbreviate "Kubernetes Engine" in general.

File: _docs/quickstarts/scanner/descriptions.md (hunk 8 above)
Comment (joecheuk) [reply _docs/quickstarts/scanner/descriptions.md]: Updated.

File: _docs/quickstarts/scanner/index.md (hunk 9 above)
Comment (joecheuk) [reply _docs/quickstarts/scanner/index.md]: Updated.

File: _docs/quickstarts/scanner/rules.md (hunk 10 above)
Comment (joecheuk) [reply _docs/quickstarts/scanner/rules.md]: Updated.

**Pull request merged**